scripts/west_commands/runners/nrf_common.py

```diff
@@ -34,6 +34,9 @@
     }
 }
 
+SUIT_STARTER_PATH = \
+Path('../modules/hal/nordic/zephyr/blobs/suit/bin/suit_manifest_starter.hex')
```

Comment:
relative to what? this looks like a bit fragile. Module abs path can be obtained by loading manifest.